src/main/scala/it/bitrock/dvs/api/model/model.scala

```diff
@@ -32,7 +32,8 @@ final case class FlightReceived(
     airline: Airline,
     airplane: Airplane,
     status: String,
-    updated: Long
+    updated: Long,
+    interpolatedUntil: Long
```

Comment:
the frontend should not notice the difference, so I believe that we should replace updated with interpolatedUntil and remove the last one

Reply:
Working on this, as soon as tests pass I'll update the PR